src/main/java/org/jabref/cli/ArgumentProcessor.java

```diff
@@ -773,6 +777,18 @@ private Optional<ParserResult> fetch(String fetchCommand) {
         }
     }
 
+    private void jumpToEntryKey(List<ParserResult> loaded, String citationKey) {
+        // search for this key in imported files
+        for (ParserResult parserResult : loaded) {
+            Optional<BibEntry> entry = parserResult.getDatabase().getEntryByCitationKey(citationKey);
+            if (entry.isPresent()) {
+                parserResult.setEntryToFocus(entry.get());
+                return;
+            }
+        }
+        LOGGER.error("Could not find given citation key to jump to");
```

Comment:
```suggestion
        LOGGER.error("Could not find citation key {} to jump to", citationKey);
```

Here, `System.out.println` is OK, because it was a CLI which outputs to the console. Please also add the output to the console. -- Maybe, even replace logger completely:

    System.out.println("Could not find citation key %0 in any library", citationKey);

See https://devdocs.jabref.org/code-howtos/localization.html for steps for the localiaztion.